crates/astria-conductor/src/sequencer/reporting.rs

```diff
@@ -0,0 +1,41 @@
+use astria_core::sequencer::v1alpha1::{
+    block::{
+        RollupTransactions,
+        SequencerBlock,
+    },
+    RollupId,
+};
+use indexmap::IndexMap;
+use serde::ser::{
+    Serialize,
+    SerializeMap as _,
+    SerializeStruct as _,
+};
+
+pub(super) struct ReportSequencerBlock<'a>(pub(super) &'a SequencerBlock);
+impl<'a> Serialize for ReportSequencerBlock<'a> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        let mut state = serializer.serialize_struct("SequencerBlockInfo", 2)?;
+        state.serialize_field("sequencer_height", &self.0.height().value())?;
+        state.serialize_field("rollups", &ReportRollups(self.0.rollup_transactions()))?;
+        state.end()
+    }
+}
+
+struct ReportRollups<'a>(&'a IndexMap<RollupId, RollupTransactions>);
```

Comment:
This reports the each rollup ID and the number of transactions per rollup. With the move to the sequencer API this will be reduced.

It could also report hashes of the actual opaque bytes. Maybe at a lower debug event level?